components/match2/commons/starcraft_starcraft2/match_group_util_starcraft.lua

```diff
@@ -269,7 +269,11 @@ function StarcraftMatchGroupUtil.constructSubmatch(games, match)
 		scores[opponentIx] = 0
 	end
 	for _, game in pairs(games) do
-		if game.winner then
+		if game.map and String.startsWith(game.map, 'Submatch') then
+			for opponentIx, score in pairs(scores) do
+				scores[opponentIx] = score + (game.scores[opponentIx] or 0)
```

Comment:
Isn't `score` and `game.scores[opponentIx]` the same?

Reply:
no, it is `scores[opponentIx]`

Reply:
Ok, some variable renames would be useful for the general cleanup.

Reply:
way more than only a few variable renames^^